types/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@joystream/types",
-  "version": "0.18.5",
+  "version": "0.18.19",
```

The idea was to bump this to v0.19.0
```suggestion
  "version": "0.19.10",
```

But you will need to update also all the other packages in the workspace that have dependency on `"^0.18.5"`  and change them to `"^0.19.0"`. This would be in
```
cli/package.json
distributor-node/package.json
query-node/mappings/package.json
storage-node/package.json
utilts/api-scripts/package.json
utils/migration-scripts/package.json
tests/package.json
```

---

@mnaamani Just FYI your suggestion uses `0.19.10` instead of `0.19.0`

---

yep fixed it thank you :+1:

---

fixed in 0c5da6f04d68bfc5f7fd85be110d4353de42680f